src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1912,9 +1912,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         if (type == Windows::Devices::Input::PointerDeviceType::Touch)
         {
+            // NB: I don't think this is correct because the touch should be in the center of the rect.
+            //     I suspect the point.Position() would be correct.

```

Comment:
followup bug?

Reply:
On GitHub? No, I don't think this is worth keeping track of, since this is just my "reviewer's thoughts". I didn't actually test whether this is a bug. I left a comment in the code because I feel like those comments will give good context if it does turn into a bug one day or maybe during a future refactor like the ones I intend to do.